demo/summer-2016.html

```diff
@@ -211,8 +211,8 @@
 <script src="kitchen-sink/require.js"></script>
 <script>
 require.config({ paths : { "ace" : "../lib/ace" } });
-require(["ace/ace", "ace/ext/language_tools", "ace/ext/papeeria/katex-previewer", "ace/ext/papeeria/highlighter"], 
-  function(ace, langTools, katexPreviewer, highlighter) {
+require(["ace/ace", "ace/ext/language_tools", "ace/ext/papeeria/katex-previewer", "ace/ext/papeeria/highlighter", "ace/ext/papeeria/highlighter_test", "ace/test/asyncjs/test"], 
```

Comment:
I don't think that we want tests to sit in this page. Ace tests, as their` Readme.md` says, can be run with `node lib/ace/test/all.js`. This currently fails because your test can't find the required modules. I don't know if we can configure paths in the test code, but I think that changing deps from e.g. `ace/edit_session` to `../../edit_session` will do the trick